diff.txt

```diff
@@ -0,0 +1,759 @@
+ ADDED      HEADER     Require-Capability:osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://java.sun.com/portlet_2_0))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/aui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/theme))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/ui))",osgi.service;filter:="(objectClass=com.liferay.blade.samples.servicebuilder.service.FooLocalService)";effective:=active,osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.7))"
+
+
+blade.strutsaction-1.0.0.jar
+*******************************************************
+CHANGED    MANIFEST   <manifest>
+ REMOVED    HEADER     Require-Capability:osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://java.sun.com/portlet_2_0))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/aui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/portlet))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/theme))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/ui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/aui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/portlet))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/theme))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/ui))",osgi.service;filter:="(objectClass=javax.servlet.ServletContext)";effective:=active,osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.7))"
+ ADDED      HEADER     Require-Capability:osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://java.sun.com/portlet_2_0))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/aui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/portlet))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/theme))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/ui))",osgi.service;filter:="(objectClass=javax.servlet.ServletContext)";effective:=active,osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.7))"
+
+
+blade.strutsportletaction-1.0.0.jar
+*******************************************************
+CHANGED    MANIFEST   <manifest>
+ ADDED      HEADER     Import-Package
+ REMOVED    HEADER     Private-Package:com.liferay.blade.samples.portlet
+ ADDED      HEADER     Private-Package:com.liferay.blade.samples.strutsportletaction
```

Comment:
gw clean not at the top level but gw clean for each subtree
and in maven you need mvn clean

I added a build-local.sh.  you can use that one to do a full clean and
rebuild before you run the diff tool

On Fri, Apr 15, 2016 at 5:26 PM, Lawrence Lee <notifications@github.com>
wrote:

> In diff.txt
> <https://github.com/liferay/liferay-blade-samples/pull/14#discussion_r59949765>
> :
>
> > + ADDED      HEADER     Require-Capability:osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://java.sun.com/portlet_2_0))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/aui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/theme))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/ui))",osgi.service;filter:="(objectClass=com.liferay.blade.samples.servicebuilder.service.FooLocalService)";effective:=active,osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.7))"
> > +
> > +
> > +blade.strutsaction-1.0.0.jar
> > +*******************************************************
> > +CHANGED    MANIFEST   <manifest>
> > + REMOVED    HEADER     Require-Capability:osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://java.sun.com/portlet_2_0))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/aui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/portlet))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/theme))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/ui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/aui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/portlet))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/theme))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/ui))",osgi.service;filter:="(objectC
>  lass=javax.servlet.ServletContext)";effective:=active,osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.7))"
> > + ADDED      HEADER     Require-Capability:osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://java.sun.com/portlet_2_0))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/aui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/portlet))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/theme))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/ui))",osgi.service;filter:="(objectClass=javax.servlet.ServletContext)";effective:=active,osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.7))"
> > +
> > +
> > +blade.strutsportletaction-1.0.0.jar
> > +*******************************************************
> > +CHANGED    MANIFEST   <manifest>
> > + ADDED      HEADER     Import-Package
> > + REMOVED    HEADER     Private-Package:com.liferay.blade.samples.portlet
> > + ADDED      HEADER     Private-Package:com.liferay.blade.samples.strutsportletaction
>
> Does gw clean do a good enough job? or do you think we actually need to
> run git clean -fdx?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/liferay/liferay-blade-samples/pull/14/files/3d16f14824811e723e7ce290e492766ed87d2717#r59949765>
>



-- 
Greg Amerson
Liferay Developer Tools
Liferay, Inc. www.liferay.com